tests/integration/tests/charm_tests/openstack_exporter.py

```diff
@@ -6,11 +6,166 @@
 import logging
 import unittest
 
+import yaml
+from charm import CLOUD_NAME, OS_CLIENT_CONFIG
+from zaza import model
+
 logger = logging.getLogger(__name__)
 
+APP_NAME = "openstack-exporter"
+SNAP_NAME = "golang-openstack-exporter"
+STATUS_TIMEOUT = 120  # must be larger than the time defined in zaza-yaml
+
+
+class OpenstackExporterBaseTest(unittest.TestCase):
+    """Base Class for charm integration tests."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Run setup for the test class."""
+        cls.leader_unit = model.get_lead_unit(APP_NAME)
+        cls.leader_unit_name = model.get_lead_unit_name(APP_NAME)
```

Comment:
It's not enough to do `cls.leader_unit.name`?

Reply:
good catch, `cls.leader_unit.entity_id` should work fine as well

Reply:
Done